### PR TITLE
init the robot before start driving

### PR DESCRIPTION
**Pull request by kzhou57**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

**Comment by kzhou57** [comment]
Looks good

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualDriveOpMode.java [hunk 1]
```diff
@@ -14,17 +14,18 @@
  * becomes relative to the field as opposed to the robot. You can
  * reset the forward heading by pressing "x".
  */
-@TeleOp(name = "Mecanum")
+@TeleOp(name = "Manual Drive")
```
Comment (kzhou57) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualDriveOpMode.java:17]: Bad move, i think it should be 
```suggestion
@TeleOp(name = "Manual Drive 2")
```

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualDriveOpMode.java [hunk 2]
```diff
@@ -14,17 +14,18 @@
...
 public class ManualDriveOpMode extends LinearOpMode {
-    private ScoopArmBot robot;
+    private ScoopArmBot robot = new ScoopArmBot(this);
```
Comment (kzhou57) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualDriveOpMode.java:19]: good !

Comment (kzhou57) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ManualDriveOpMode.java:19]: No, it's not good

**Review by rzhang57 — APPROVED** [review]
good job

**Pull request merged**